content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -675,6 +687,16 @@ This endpoint unlocks a workspace.
 | --------------- | ------------------------------------------------------------------------------------------------------------- |
 | `:workspace_id` | The workspace ID to unlock. Obtain this from the [workspace settings](../workspaces/settings.html) or the [Show Workspace](#show-workspace) endpoint. |
 
+Status  | Response                                     | Reason(s)
+--------|----------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "workspaces"`) | Successfully unlocked the workspace
+[404][] | [JSON API error object][]                    | Workspace not found, or user unauthorized to perform action
```

Comment:
As of your PR, it just 200s if the workspace is not already locked.

Reply:
409s for unlock/force-unlock will be introduced in a later PR.

Also, when you try to lock a workspace that you have already locked, it will 200. This will be changed to 409 in a later PR to TFE.

Reply:
Ah! I misread the code. 👍